dependencies/meson.build

```diff
@@ -0,0 +1,51 @@
+# nix dependencies build file
+#============================================================================
+
+
+# Required dependencies
```

Comment:
I think these comments are superfluous - the textbook case of

`i = i+1; /* adds 1 to i */`

Reply:
it's overly verbose in order to allow people to more easily understand the new buildsystem, especially if they're not familiar with meson :)

Reply:
Yep, as a beginner in Meson I really liked the plenty of comments. 

But I believe in the school of self-documenting code too. 
And `required_dep_dirs` already conveys the idea of "required dependencies".

Reply:
fair point :)